### ISSUE TITLE
download_resources.ipynb

### ISSUE THREAD

**Issue by zuenko** [hse-aml/intro-to-dl#4]
All resources: "Download failed"

**Comment by ZEMUSHKA** [comment]
@zuenko 
I've updated the code (commit cd53c2ab0), could you try it and print the errors you see? 
Also, what OS do you have?

**Comment by zuenko** [comment]
@ZEMUSHKA That night i solved it, the problem was in packages  conda_requirements.txt. 
And I've got some recommendation for docker-users. 
I've my own docker-jupyter and jupyter_contrib_nbextensions won't install if you with enabled notebook, so i recommend to use  pip [Installation docks](http://jupyter-contrib-nbextensions.readthedocs.io/en/latest/install.html#install-the-python-package) , and then enable it with: 
`jupyter nbextension enable --py --sys-prefix widgetsnbextension`
`jupyter labextension install @jupyter-widgets/jupyterlab-manager`
or without '--sys-prefix'

And now it's okay, but I'd like to recommend not to use that:
`pip install -r requirements.txt` ,
Because there are problems with double packages:
![image](https://user-images.githubusercontent.com/17703283/34919423-d1c21892-f974-11e7-996c-b76508470d7d.png)
And now i need to solve that.

To sum up:
1. Don't forget to install  conda_requirements.txt
2. If you use your own docker install it without jupyter_contrib_nbextensions and then use instructions above.

P.s. And adding exceptions in download_resources.ipynb which describes package missing will be great

**Comment by ZEMUSHKA** [comment]
@zuenko 
You've used `conda_requirements.txt` and `pip` at the same time. The first is supposed to be used with Anaconda (our recommended setup for Windows without Docker) and the second with our Docker container which doesn't use Anaconda at all.

Sorry if that was confusing.

Just to make it clear what options we recommend:
1) Either use Docker or install missing parts following our Dockerfile (https://github.com/ZEMUSHKA/coursera-aml-docker/blob/master/Dockerfile) on clean Ubuntu  which uses https://github.com/ZEMUSHKA/coursera-aml-docker/blob/master/requirements.txt for `pip` installation.
2) If you have Anaconda (in Docker or anywhere else) or have just installed it, you should follow these instructions: https://github.com/hse-aml/intro-to-dl#anaconda-option-best-for-windows

P.S. We now have better exception handling, so it should be more clear what's going on.
